bot/src/infrastructure/monitoring/README.md

```
@@ -0,0 +1,412 @@
+# Monitoring System
+
+Comprehensive monitoring, metrics collection, and distributed tracing for the EVE Online Discord Bot.
+
+## Features
+
+### Health Checks
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)